src/titiler/core/titiler/core/dependencies.py

```diff
@@ -320,6 +320,20 @@ def __post_init__(self):
             self.max_size = None
 
 
+@dataclass
+class PartFeatureParams(DefaultDependency):
```

Comment:
New Dependency for `/bbox`, `/feature` and `/statistics [POST]` endpoint where we don't set `max_size` default

Reply:
`PartFeatureParams` is a confusing name but I can't really thing of anything better myself 🤷